src/asset_library.rs

```diff
@@ -454,7 +454,7 @@ impl MintInfo {
 /// Indexable collection of asset types.
 #[derive(Clone, Debug, Default, PartialEq)]
 pub struct AssetLibrary {
-    assets: Vec<AssetInfo>,
```

Comment:
Why make this public? There is also `pub fn AssetLibrary::iter`

Reply:
Good catch, I don't know why I did this initially but I reverted this change.